### ISSUE TITLE
Checks are not ignoring databases listed in parameter -ExcludeDatabase

### ISSUE THREAD

**Issue by gbargsley** [sqlcollaborative/dbachecks#341]
-ExcludeDatabase ReportServer, ReportServerTempDB is not skipping the databases for all the tests.

Example of failed test.
![image](https://user-images.githubusercontent.com/169602/36817639-75113186-1ca7-11e8-9662-e12cd4b64e4c.png)

**Comment by js0505** [comment]
I can confirm that this is not working in the current release 1.0.5 as expected.

**Comment by js0505** [comment]
-ExcludeDatabase does not seem to be functioning at all anywhere.

**Comment by js0505** [comment]
Executed Invoke-DbcCheck -SqlInstance localhost -AllChecks -ExcludeDatabase ReportServer and it still shows up on all of the tests

**Comment by SQLDBAWithABeard** [comment]
Yes, there are some checks that I have noticed that are missing the capability to exclude
